### ISSUE TITLE
Need a custom template for p-paginator to provide accessibility to the page links, and buttons.

### ISSUE THREAD

**Issue by VigneshVaidyanathan** [primefaces/primeng#11601]
### Describe the feature you would like to see added

I am working on a project that needs to implement accessibility on the pagination items for the table. We are using the p-table which internally uses the p-paginator to render the page links. I want to add things like aria-label, tabIndex etc with respect to accessibility on those.

Is there a way to set accessibility attributes for paginator? 
Is it possible to send a custom template to the paginator, which we can add and manage the accessibility that it needs?

### Is your feature request related to a problem?

I am facing this issue when trying to implement accessibility in the pagination controls that are displayed on the table.

### Describe the solution you'd like

The solutions that we have discussed are
1. Query the DOM, get hold of the page links rendered and update the properties. Problem is this has to be rendered everything a page is clicked, as it could bring in new items into the page links when the pages move right.
2. Allow templating for the paginator similar to p-table, which allows us to customize the DOM that we give it.

**Comment by cetincakiroglu** [comment]
Hi,
We're fully focused on Accessibility. Thanks so much for sharing these with us.
Our main goal is to consider all the components one by one and document the individual accessibility features. In doing so, it is to correct the deficiencies or faulty implementations of that component. For this reason, our team carries out a comprehensive and meticulous study. Of course, it will take some time to complete this process.
Currently, working on the components specified in your attached documentation will indeed be a temporary solution for us and we cannot guarantee that these solutions will not change. Because we examine a lot of accessibility documents and try to make the best decisions for users. At this stage, the best decision that can be made about such issues is to stop the actions to be taken on them for the time being. We will reactivate these issues after we complete our work on accessibility in Prime* component libraries. In this way, we will be able to make better decisions about issues.
Details; https://www.primefaces.org/primeng/accessibility
Thanks a lot for your understanding! Best Regards,

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
